src/components/list/list.html

```diff
@@ -0,0 +1 @@
+<ng-content></ng-content>
```

Comment:
Yeah, agree. Originally I separated it in case it got any larger, but it didn't. Inlining.